Rubberduck.Core/Common/CodeString.cs

```diff
@@ -69,18 +68,10 @@ public static CodeString FromString(string code)
         /// One-based position of the code string in the containing module.
         /// </summary>
         public Selection SnippetPosition { get; }
-
-        public string[] Lines
-        {
-            get
-            {
-                return Code?.Split('\n') 
-                    ?? new string[] { };
-            }
-        }
-
-        public static bool operator ==(CodeString codeString1, CodeString codeString2) => (codeString1.Code == codeString2.Code && codeString1.CaretPosition == codeString2.CaretPosition);
-        public static bool operator !=(CodeString codeString1, CodeString codeString2) => !(codeString1 == codeString2);
+        /// <summary>
+        /// Gets the individual <see cref="Code"/> string lines.
+        /// </summary>
+        public string[] Lines => Code?.Split('\n') ?? new string[] { };
```

Comment:
What about the `\r`?

Reply:
it's effectively dangling. didn't think it mattered, since I'm re-joining later using `\n`... or does it?

Reply:
done